### PR TITLE
Confirm At Least 1 AMI Chart #172512100

### PR DESCRIPTION
**Pull request by LoganArnett**
This PR is for ticket [#172512100](https://www.pivotaltracker.com/story/show/172512100)

### CODE REVIEW

File: app/javascript/components/supplemental_application/sections/ConfirmedHouseholdIncome.js [hunk 1]
```diff
@@ -51,22 +51,23 @@ const ConfirmedHouseholdIncome = ({ listingAmiCharts, form, visited }) => {
 
   useEffect(() => {
     const getAmiCharts = async () => {
-      let chartTypes = []
       let amis = await getAmis(listingAmiCharts, totalHouseholdSize)
```
Comment (akegan) [inline app/javascript/components/supplemental_application/sections/ConfirmedHouseholdIncome.js]: we should be able to remove this query entirely depending on what's available in listingAmiCharts already.

**Review by akegan — CHANGES_REQUESTED** [review]
I think we can remove an excess API call here now before we merge

File: app/javascript/components/supplemental_application/sections/ConfirmedHouseholdIncome.js (hunk 1 above)
Comment (LoganArnett) [reply app/javascript/components/supplemental_application/sections/ConfirmedHouseholdIncome.js]: I pushed a removal here and found that the getAmis was only used here and in a file testing the function so I removed those instances as well

File: app/javascript/components/supplemental_application/sections/ConfirmedHouseholdIncome.js [hunk 2]
```diff
@@ -43,19 +29,17 @@ export const getAmiPercent = ({income, ami}) => {
...
   const [ selectedType, setSelectedType ] = useState(currentAmiChartType || '')
 
   useEffect(() => {
     const getAmiCharts = async () => {
-      let amis = await getAmis(listingAmiCharts, totalHouseholdSize)
-
-      if (amis.length > 0) {
-        let chartTypes = amis.map((chart) => {
-          const value = `${chart.name} - ${chart.year}`
+      console.log(listingAmiCharts)
```
Comment (akegan) [inline app/javascript/components/supplemental_application/sections/ConfirmedHouseholdIncome.js]:
```suggestion
```

**Review by akegan — APPROVED** [review]
Need to remove a stray console.log, but otherwise looks good. You can ignore the circleCI builds for now.

**Comment by akegan** [comment]
test comment for slack integration

**Comment by akegan** [comment]
another test comment for slack integration

**Pull request merged**